docs/pages/web/textfield.js

```diff
@@ -36,6 +39,35 @@ function Example(props) {
   );
 }
 `}
+        slimBanner={
+          <SlimBanner
+            iconAccessibilityLabel="Component under experiment"
+            message={`Textfield is under an experiment to improve its password typing UI: adds an "show/hide password" icon button for type="password".`}
+            type="warning"
+            helperLink={{
+              text: 'Visit the Pull Request to learn more.',
+              accessibilityLabel: '',
+              href: 'https://github.com/pinterest/gestalt/pull/1995',
+              onClick: () => {},
+              target: 'blank',
+            }}
+            primaryAction={{
+              accessibilityLabel:
+                experiments === generatedDocGen?.displayName
+                  ? 'Deactivate component experiments in the Docs'
+                  : 'Activate component experiments in the Docs',
+              label:
+                experiments === generatedDocGen?.displayName
+                  ? 'Deactivate experiments'
+                  : 'Activate experiments',
+              onClick: () =>
+                setExperiments(
+                  experiments === generatedDocGen?.displayName ? '' : generatedDocGen?.displayName,
```

Comment:
It looks like this is all-or-nothing, yes? We should provide users more granularity here. Users should be able to toggle individual experiments.

Reply:
I tried stringifying an array of experiments but it couldn't get to work. It's not like we have >1 experiments at the time for each single component. This is a first iteration of this implementation. With cmp name I hit both desktop and web experiments in a clean way. 

If there's need for refactoring this to allow overlapping experiments we can rethink the implementation.

Reply:
What didn't work about stringifying an array or object of experiments?